pkg/unpack/unpacker.go

```diff
@@ -320,7 +320,9 @@ func (u *Unpacker) unpack(
 		for try := 1; try <= 3; try++ {
 			// Prepare snapshot with from parent, label as root
 			key = fmt.Sprintf(snapshots.UnpackKeyFormat, uniquePart(), chainID)
+			ctx, prepareSpan := tracing.StartSpan(ctx, tracing.Name(unpackSpanPrefix, "prepare"))
```

Comment:
Tiny nit: do we want to override `ctx` all along? I'm not sure it changes much but maybe for span visibility (so that they do not appear all as the child of the previous one) it would be slightly better?

Reply:
very good point [c692a73](https://github.com/DataDog/containerd/pull/9/commits/c692a737514dbaee557b39e8d3d556f06f04b787)